src/DataSource/Interpreter/JsonFileInterpreter.php

```diff
@@ -52,7 +67,14 @@ protected function doInterpretFileAndCallProcessRow(string $path): void
 
     public function setSettings(array $settings): void
     {
-        //nothing to do
+        $path = $settings['path'];
+        try {
+            (new Parser)->parse($path);
```

Comment:
we should allow empty paths here as well...

Reply:
this has been addressed.